rpcs3/util/serialization.hpp

```diff
@@ -82,6 +82,48 @@ namespace utils
 			return true;
 		}
 
+		template <bool IsWriting, typename T>
```

Comment:
Does it really support args of different size? What about u128?

Reply:
Added support.